firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -234,7 +228,7 @@ filament_diameter: 1.75
 #! In HE0 Position (Hotend heater)
 heater_pin: PB1
 ## Validate the following thermistor type to make sure it is correct
-sensor_type: ATC Semitec 104GT-2
+#sensor_type: ATC Semitec 104GT-2
```

Comment:
can you add `## See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options` underneath this line to point users to the list of thermistor types available. (maybe for bed heater too, but it doesn't really matter.